README.md

```diff
@@ -1,104 +1,99 @@
-- [Official documentation for installing rbenv](https://github.com/rbenv/rbenv?tab=readme-ov-file#installation)
+<details>
+  <summary>Instructions for running from source</summary>
+  ℹ️ Crawler uses both JRuby and Java.
+  We recommend using version managers for both.
+  When developing Crawler we use <b>rbenv</b> and <b>jenv</b>.
+  There are instructions for setting up these env managers here:
 
-Go to the root of the Crawler directory and check the expected Java and Ruby versions are being used:
+  - [Official documentation for installing jenv](https://www.jenv.be/)
+  - [Official documentation for installing rbenv](https://github.com/rbenv/rbenv?tab=readme-ov-file#installation)
 
-```bash
-# should output the same version as `.ruby-version`
-$ ruby --version
+  1. Clone the repository
+  2. Go to the root of the Crawler directory and check the expected Java and Ruby versions are being used:
+      ```bash
+      # should output the same version as `.ruby-version`
+      $ ruby --version
 
-# should output the same version as `.java-version`
-$ java --version
-```
+      # should output the same version as `.java-version`
+      $ java --version
+      ```
 
-If the versions seem correct, you can install dependencies:
+  3. If the versions seem correct, you can install dependencies:
+      ```bash
+      $ make install
+      ```
 
-```bash
-$ make install
-```
-
-You can also use the env variable `CRAWLER_MANAGE_ENV` to have the install script automatically check whether `rbenv` and `jenv` are installed, and that the correct versions are running on both:
-Doing this requires that you use both `rbenv` and `jenv` in your local setup.
+     You can also use the env variable `CRAWLER_MANAGE_ENV` to have the install script automatically check whether `rbenv` and `jenv` are installed, and that the correct versions are running on both:
+     Doing this requires that you use both `rbenv` and `jenv` in your local setup.
 
-```bash
-$ CRAWLER_MANAGE_ENV=true make install
-```
-
-Crawler should now be functional.
-See [Configuring Crawlers](#configuring-crawlers) to begin crawling web content.
+      ```bash
+      $ CRAWLER_MANAGE_ENV=true make install
+      ```
+</details>
 
 ### Configuring Crawlers
 
 See [CONFIG.md](docs/CONFIG.md) for in-depth details on Crawler configuration files.
 
-Once you have a Crawler configured, you can validate the domain(s) using the CLI.
-
-```bash
-$ bin/crawler validate config/my-crawler.yml
-```
-
-If you are running from docker, you will first need to copy the config file into the docker container.
-
-```bash
-# copy file (if you haven't already done so)
-$ docker cp /path/to/my-crawler.yml crawler:config/my-crawler.yml
-
-# run 
-$ docker exec -it crawler bin/crawler validate config/my-crawler.yml
-```
-
-See [Crawling content](#crawling-content).
+### CLI Commands
 
-### Crawling content
+Open Crawler has no UI.
+All interactions with Crawler take place through the CLI.
```

Comment:
general remark: I'd be consistent in usage of Open Crawler for the tool's name. Can use "a crawler" to refer to a specific instance.